pkgs/development/libraries/flann/default.nix

```diff
@@ -1,24 +1,58 @@
-{ lib, stdenv, fetchFromGitHub, fetchpatch, unzip, cmake, python }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, unzip
+, cmake
+, python ? null
+, lz4
+, pkg-config
+, enablePython ? false
+}:
+
+assert enablePython -> python != null;
```

Comment:
```suggestion
}:
```

Reply:
Is there guidance on doing or not-doing this? I saw it used in some other packages and just mindlessly copied, eg:

https://github.com/NixOS/nixpkgs/search?q=assert+enablePython

Reply:
Yes, those are all wrong and using an old technique that is not required and can cause issues with overwrites.